openfisca_us/tests/policy/baseline/gov/states/md/tax/income/credits/md_ctc.yaml

```
@@ -0,0 +1,22 @@
+- name: Two eligible children.
+  period: 2020
+  input:
+    people:
+      head:
```

And delete unused characteristics
```suggestion
      head: {}
```